### PR TITLE
Py3.11 fixes

### PR DESCRIPTION
**Pull request by karstenw**
Some fixes for Python 3.11 and numpy 1.25

### CODE REVIEW

File: brandmateriel/+latex/brandmateriel.tex [hunk 1]
```diff
@@ -0,0 +1,51 @@
+\documentclass[10pt]{article}
```
Comment (skymandr) [inline brandmateriel/+latex/brandmateriel.tex:1]: I like `LaTeX`, but I'm not sure what the purpose of this file is?

File: brandmateriel/src/engine/shader.py [hunk 2]
```diff
@@ -182,7 +182,7 @@ def linear_distance(self, val):
         self._linear_distance = val
 
     def apply_lighting(self, positions, normals, colours, culling=True,
-                       scatter=True, fading=True):
+                       scatter=True, fading=False):
```
Comment (skymandr) [inline brandmateriel/src/engine/shader.py]: Why this change?

File: brandmateriel/src/game.py [hunk 3]
```diff
@@ -636,7 +636,7 @@ def do_step(self, surface):
             try:
                 self._gameover = (
                     self._gameover_timer
-                    < dt.datetime.now() - dt.timedelta(seconds=1)
+                    < dt.datetime.now() - dt.timedelta(seconds=10)
```
Comment (skymandr) [inline brandmateriel/src/game.py]: Why this change? (I honestly can't remember what the `_game_over_timer` does...)

**Review by skymandr — COMMENTED** [review]
Grateful for your continued interest in the code! I should really run a formatter on it, but in case you have any other changes in the works, that would lead to horrible merge conflicts....

Anyway, looks good, but a few questions that I would appreciate your comments to before merging.

File: brandmateriel/+latex/brandmateriel.tex (hunk 1 above)
Comment (karstenw) [reply brandmateriel/+latex/brandmateriel.tex:1]: Can be removed. It's a thing I use for source code pretty printing.

I think you should `pdflatex` it, look at the result and decide for yourself to toss or keep it.

File: brandmateriel/src/engine/shader.py (hunk 2 above)
Comment (karstenw) [reply brandmateriel/src/engine/shader.py]: I don't remember. May be a leftover of times tinkering with the code.

File: brandmateriel/src/game.py (hunk 3 above)
Comment (karstenw) [reply brandmateriel/src/game.py]: I don't remember either. My guess that's the play time after player death. Seems to be from tinkering or bug hunting. Definitely forgot to set it back.

Comment (karstenw) [reply brandmateriel/src/game.py]: Just watched it. That was a tangent I did not follow further. The game engine does funny stuff with the ship remains like bouncing in mid air.

**Comment by karstenw** [comment]
Sorry for the inconveniences; I thought I had my act cleaned up much better that it was.

File: brandmateriel/src/game.py (hunk 3 above)
Comment (skymandr) [reply brandmateriel/src/game.py]: Interesting. I wonder why it does that. I probably put that timer in place because I couldn't figure it out...

File: brandmateriel/+latex/brandmateriel.tex (hunk 1 above)
Comment (skymandr) [reply brandmateriel/+latex/brandmateriel.tex:1]: I always like a LaTeX-document. I have too few in my life at the moment. :)

File: brandmateriel/src/engine/shader.py (hunk 2 above)
Comment (skymandr) [reply brandmateriel/src/engine/shader.py]: I'll revert this then.

File: brandmateriel/src/game.py (hunk 3 above)
Comment (skymandr) [reply brandmateriel/src/game.py]: I'll revert this.

**Comment by skymandr** [comment]
> Sorry for the inconveniences; I thought I had my act cleaned up much better that it was.

I feel that I should be the one apologising for the inconvenience: for writing such spaghetti code and then not maintaining it properly. I appreciate that you have taken the time to update it! ;)

I'm not sure if you feel you want to continue playing with it. Do you have an opinion on if I should run an automatic formatter on it?

File: brandmateriel/src/game.py (hunk 3 above)
Comment (skymandr) [reply brandmateriel/src/game.py]: Ah, you've already done it. Thanks!

**Review by skymandr — APPROVED** [review]

**Comment by karstenw** [comment]
> I'm not sure if you feel you want to continue playing with it. Do you have an opinion on if I should run an automatic formatter on it?

No. I think it's OK in golang where the true-one-and-only style was there from the beginning; but everywhere else formatting is an act that needs time, energy and thought.

Sometimes I reformat code just to better understand it or to reflect my current understanding.

So my code will always reflect & document my stupidity and my fight with it.

I guess that's opinionated....

EDIT: I am against automatic formatters (except go) but like me a good enough pretty printer.
